### ISSUE TITLE
How is the license determined?

### ISSUE THREAD

**Issue by ErikAtApex** [ros-visualization/rqt_publisher#45]
On [the ROS wiki](https://wiki.ros.org/rqt_publisher) and [ROS index](https://index.ros.org/p/rqt_publisher/), the rqt_publisher license is listed as `BSD`. In this repository, I can't find a license file or any mention of the license. How are these sites determining the license? Would it be appropriate to add an explicit license file to this repository?

Thanks!

**Comment by clalancette** [comment]
You are right that this should have a LICENSE file.  See #46, which adds this in.

**Comment by sloretz** [comment]
Fixed by #46
